mypy/build.py

```diff
@@ -1896,49 +1883,54 @@ def parse_file(self) -> None:
         # TODO: Why can't SemanticAnalyzerPass1 .analyze() do this?
         self.tree.names = manager.semantic_analyzer.globals
 
+        for pri, id, line in manager.all_imported_modules_in_file(self.tree):
+            if id == '':
```

Comment:
It's not obvious why an empty string indicates a relative import. Add a longer description, probably here and in the docstring of `all_imported_modules_in_file`.